app/controllers/api/v1/screening_lists/consolidated_controller.rb

```diff
@@ -1,4 +1,4 @@
 class Api::V1::ScreeningLists::ConsolidatedController < ApiController
   include Searchable
-  search_by :countries, :q, :type, :sources, :name, :fuzziness, :address
+  search_by :countries, :q, :type, :sources, :name, :distance, :address
```

Comment:
Generally, we try not to change the v1 interface, but maybe in this case the "fuzziness" feature isn't documented and hence isn't in active use..?